src/models/currentUser.js

```diff
@@ -51,6 +51,7 @@ export const newestMembership = currentUser =>
   maxBy(get('memberships', currentUser), 'created_at')
 
 export const hasFeature = (currentUser, key) => {
+  if (process.env.NODE_ENV === 'test') return true
```

Comment:
i don't think we want to assume that the test environment has all features turned on. for example we might want at some point to test that a page renders correctly with and without a given feature enabled.